CHANGES-4.3.md

```diff
@@ -21,6 +21,8 @@ File format:
 
 * Fix case where publishing to a non-existent topic alias would crash the connection [#6979]
 * Fix HTTP-API 500 error on querying the lwm2m client list on the another node [#7009]
+* Fix Server-KeepAlive wrongly applied on MQTT v3.0/v3.1 [#7085]
```

Comment:
Bad diff?

Reply:
For https://github.com/emqx/emqx/pull/7085